assets/scss/6-components/table/_table.scss

```diff
@@ -1,23 +1,68 @@
+@mixin fixed-table-widths($widths) {
+  @for $i from 1 through length($widths) {
+    $curr: nth($widths, $i);
+
+    thead tr:first-child > th:nth-child(#{$i}),
+    tbody tr:first-child > td:nth-child(#{$i}) {
+      width: $curr;
+    }
+  }
+}
+
 // Table (c-table)
 //
-// Helper for table display
+// Tabular display classes
 //
 // .c-table--striped - Zebra stripes odd rows
+// .c-table--bordered - Bordered rows
+// .c-table--10-45-45 - Three-column table with col widths 10%, 45%, 45%
+// .c-table--15-70-15 - Two-column table with col widths 15%, 70%, 15%
 //
 // Markup: 6-components/table/table.html
 //
 // Styleguide 6.1.5
 .c-table {
+  // @extend .font-sansserif; // use font helper
   border-collapse: collapse;
+  width: 100%; // use helper class
```

Comment:
should we go ahead and remove `width: 100%; ` since we have that helper? And the comment of extending the font

Reply:
Gone, sorry. Cruft I forgot to delete.